A41-xds-rbac.md

```diff
@@ -150,7 +150,7 @@ In RBAC `metadata` refers to the Envoy metadata which has no relation to gRPC
 metadata. Envoy metadata is generic state shared between filters, which has no
 gRPC equivalent. RBAC implementations in gRPC will treat Envoy metadata as an
 empty map. Since `ValueMatcher` can only match if a value is present (even
-`NullMatch`), the `metadata` matcher is guaranteed not to match.
+`NullMatch`), it is guaranteed not to match. If `invert` is set, it will match.
```

Comment:
I'm not wild about special-casing metadata matcher for describing invert behavior. I _know_ someone will complain to me saying, "but invert was only mentioned for 'metadata', so I thought it only applied there". If we need to call out invert behavior, make it separate because it applies to all parts. Include it near the "Because matching supports NOT, the matcher must still be processed even if a rule contains references to things that don't generally match" text?

Reply:
Updated